hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/ContainerUtils.java

```diff
@@ -341,4 +344,16 @@ public static long getPendingDeletionBlocks(ContainerData containerData) {
               " not support.");
     }
   }
+
+  public static void assertSpaceAvailability(long containerId, HddsVolume volume, int sizeRequested)
+      throws StorageContainerException {
+    final SpaceUsageSource currentUsage = volume.getCurrentUsage();
+    final long spared = volume.getFreeSpaceToSpare(currentUsage.getCapacity());
+
+    if (currentUsage.getAvailable() - spared <= sizeRequested) {
```

Comment:
@sumitagrawl , It should be `<` since `==` means that the space is enough.

Reply:
updated